filter/string_match.go

```diff
@@ -0,0 +1,89 @@
+package filter
+
+import (
+	"bytes"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/AdRoll/baker"
+)
+
+// StringMatchDesc describes the StringMatch filter
+var StringMatchDesc = baker.FilterDesc{
+	Name:   "StringMatch",
+	New:    NewStringMatch,
+	Config: &StringMatchConfig{},
+	Help:   "Discard records if a field matches any of the provided strings",
+}
+
+// StringMatchConfig holds config parameters of the StringMatch filter.
+type StringMatchConfig struct {
+	Field   string   `help:"name of the field which value is used for string comparison" required:"true"`
+	Strings []string `help:"list of strings to match." required:"true"`
+	Invert  bool     `help:"Invert the outcome, so that records are discarded if they don't match any of the strings" default:"false"`
```

Comment:
super-nit: maybe I'd call it `InvertMatch`

Reply:
done